networkx/classes/graph.py

```diff
@@ -852,14 +854,15 @@ def add_edges_from(self, ebunch, attr_dict=None, **attr):
         >>> G.add_edges_from([(3,4),(1,4)], label='WN2898')
         """
         # set up attribute dict
+        edge_attr = {}
         if attr_dict is None:
```

Comment:
Can we do a similar change to this structure as with my comment above? Use ```if attr_dict is not None:``` and proceed from there?